src/notifications/__init__.py

```diff
@@ -59,7 +65,14 @@ def _newLobby(self, game):
 
         self.events.append((self.NEW_GAME, game.copy()))
         self.checkEvent()
-
+		
+    def _gamefull(self):
+        if self.isDisabled() or not self.settings.popupEnabled(self.GAME_FULL):
+            return
+        notify_mode = self.settings.getCustomSetting(self.GAME_FULL, 'mode')
+        self.events.append((self.GAME_FULL, "game full ?"))
```

Comment:
Second argument is optional data for an event handler. You're not using it, so I suggest using a None as the second argument.